base/loading.jl

```diff
@@ -1757,11 +1757,15 @@ associated cache is relocatable.
 """
 function isrelocatable(pkg::PkgId)
     path = compilecache_path(pkg)
+    @show pkg
```

Comment:
This PR adds a bunch of `@show`s, also below and in other files, presumably they should be removed?

Reply:
Yes. These are leftovers from debugging CI, I just did not have time yet to rebase.